WebService/Startup.cs

```diff
@@ -18,6 +18,7 @@
 using Microsoft.Extensions.Logging;
 using Newtonsoft.Json;
 using ILogger = Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Diagnostics.ILogger;
+using Microsoft.AspNetCore.Mvc;
```

Comment:
Do we know why this is needed?

Reply:
This is needed for the changes in line 75 (as per documentation for net core 2.1 upgrade) -

services.AddMvc().AddControllersAsServices().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);